044-DevOpswithGithubActions/Coach/README.md

```diff
@@ -0,0 +1,32 @@
+# Welcome to Devops With GitHub Hackathon
+
+## Overview
```

Comment:
This "overview" section and the image repeats the content on the hack's homepage.  It is redundant and should be removed from the Coach's Guide "table of contents" (ToC) page.

For all coach guides, 
Please add challenge name to each coach guide page as per the challenge-solution-template in the WTH author's guide:
https://github.com/microsoft/WhatTheHack/blob/master/000-HowToHack/WTH-Challenge-Solution-Template.md

Example:
"Challenge 01: Assessment and Getting Ready for Prerequisites - Coach's Guide"

This example included inline in the solution for Challenge 1.